.github/workflows/e2e/k8s/metric/telemetrygen-statefulset.yml

```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata: 
+  name: telemetrygen-metric-statefulset
+  namespace: default
+spec:
+  serviceName: telemetrygen-metric-statefulset
+  replicas: 1
+  selector:
+    matchLabels:
+      app: telemetrygen-metric-statefulset
+  template:
+    metadata:
+    metadata:
+      annotations:
+        workload: statefulset
+      labels:
+        app: telemetrygen-metric-statefulset
+    spec:
+      containers:
+      - command:
+        - /telemetrygen
+        - metrics
```

Comment:
Can we turn these repetitive files into templates? e.g replace `metrics` with `${DATA_TYPE}` here and call

```
export DATA_TYPE="metrics"
envsubst < .github/workflows/e2e/k8s/trace/telemetrygen-daemonset.yml | kubectl -n default create -f -
```

instead of
```
kubectl -n default create -f .github/workflows/e2e/k8s/trace/telemetrygen-daemonset.yml
```

This env var substitution technique should work, but any other templating solution would be good as well.

I just think it'll be easier to maintain, in case we need to make a change in them going forward.

Reply:
good point, I will try.

Reply:
Done, I used `sed` instead. @dmitryax